imports/client/components/SetupPage.tsx

```diff
@@ -75,8 +75,9 @@ enum SubmitState {
 
 const googleCompletenessStrings = [
   'Unconfigured',
-  '1/3 complete',
-  '2/3 complete',
+  '1/4 complete',
+  '2/4 complete',
+  '3/4 complete',
```

Comment:
Do we intend to nudge people towards making a root folder, or is it more of an optional thing?  I guess we nudge them towards having a template doc, but I feel like that's more impactful to user experience than making sure that you're putting data in a subdirectory instead of your drive root.

I guess I'm wondering if gdrive.root this should be an optional setting?  If I really do want to use the drive root (e.g. because we have a whole account for running jolly-roger as), it looks like the only way to achieve that is to leave gdrive.root unconfigured, but then we'll display this config as "incomplete", which seems confusing.

Reply:
Yeah that's reasonable. I was riffing off of the behavior of the template documents, but I can treat this as optional